src/alt/store/StoreMixin.js

```diff
@@ -19,69 +19,38 @@ const StoreMixin = {
     this.dispatcher.waitFor(tokens)
   },
 
-  exportAsync(asyncMethods) {
-    this.registerAsync(asyncMethods)
-  },
-
-  registerAsync(asyncDef) {
-    let loadCounter = 0
-
-    const asyncMethods = fn.isFunction(asyncDef)
-      ? asyncDef(this.alt)
-      : asyncDef
-
-    const toExport = Object.keys(asyncMethods).reduce((publicMethods, methodName) => {
-      const desc = asyncMethods[methodName]
-      const spec = fn.isFunction(desc) ? desc(this) : desc
+  fetch(spec) {
```

Comment:
if we want to merge this in now I should bring back registerAsync and exportAsync and also add fetch. But deprecate the former two.